test/alternative_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "spec_helper.rb"
+
+describe "UpdateAlternatives::Alternative" do
+
+  context "when we get all alternatives names" do
```

Comment:
no method description here. I expect something like `describe ".all_names" do` and then ` it "returns an array with all alternatives names" do`. Please do not use should in specification. Specification say how it works, not how it can possible work.

Still my note from above code also apply. Do we want all_names to be public method? I do not see use case for it. In general when I design API I try to make minimal API that make sense and then extend it if needed by use case.